### PR TITLE
Patch the ability to use SELinux with setup_repo disabled

### PR DESCRIPTION
**Pull request by zwall-bp**
# Description

This is an addendum to the #355 patch, where the path to the SELinux module is different, based on the version of MariaDB installed. SELinux was not tested during the initial development.

## Issues Resolved

* Developers who disable `setup_repo` on an SELinux-enabled node will no longer suffer from a converge error.

## Check List

- [ ] A summary of changes made is included in the CHANGELOG under `## Unreleased`
- [ ] New functionality includes testing.
- [ ] New functionality has been documented in the README if applicable.

### CODE REVIEW

**Review by ramereth — CHANGES_REQUESTED** [review]
Please update `CHANGELOG.md`

File: CHANGELOG.md [hunk 1]
```diff
@@ -6,6 +6,10 @@ This project adheres to [Semantic Versioning](http://semver.org/).
 
 ## Unreleased
 
+## 5.2.19 - *2023-04-18*
```
Comment (ramereth) [inline CHANGELOG.md]: Please do not add a new release. This will be handled automatically by our release system.

**Review by ramereth — CHANGES_REQUESTED** [review]

File: CHANGELOG.md (hunk 1 above)
Comment (zwall-bp) [reply CHANGELOG.md]: Oh, so put it under `## Unreleased` instead?

Comment (ramereth) [reply CHANGELOG.md]: @sectopodwreck yes please

File: CHANGELOG.md [hunk 2]
```diff
@@ -6,6 +6,8 @@ This project adheres to [Semantic Versioning](http://semver.org/).
 
 ## Unreleased
 
+Patched the ability to use SELinux with `setup_repo` disabled
```
Comment (ramereth) [inline CHANGELOG.md]:
```suggestion
- Patched the ability to use SELinux with `setup_repo` disabled
```

**Review by ramereth — APPROVED** [review]

**Comment by kitchen-porter** [comment]
Released as: [5.2.19](https://github.com/sous-chefs/mariadb/releases/tag/5.2.19)

**Pull request merged**